File: src/main/java/com/example/librarymanagement/dto/JournalDto.java
package com.example.librarymanagement.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class JournalDto {
    private Long id;
    private LocalDate dateOfBorrowing;
    private LocalDate dateOfReturning;
    private String title;
    private String authorName;
    private String user;
    private String nameOfLibrary;
}

File: src/main/java/com/example/librarymanagement/dto/JournalDto.kt
package com.example.librarymanagement.dto

import java.time.LocalDate

data class JournalDto (
    val id: String?,
    val dateOfBorrowing: LocalDate?,
    val dateOfReturning: LocalDate? = null,
    val bookPresenceId: String,
    val userId: String,
)
